src/concepts/analysis_gotchas.md

```diff
@@ -126,25 +126,26 @@ ask them to install Beta.
 
 ### Trusting Dates
 
-Don't trust client times.
+Don't trust desktop client times. The situation is somewhat better on mobile devices
+and being investigated.
```

Comment:
```suggestion
and is being investigated.
```
If there's a bug for the investigation, here'd be a good place to link to it.

Reply:
I knew you'd ask this. There is not a specific bug, but this is generally an important area for understanding day-by-day retention in Fenix that is getting a lot of interest right now.